### PR TITLE
improvement(test_rolling_upgrade.py): Creating upgrade rolling for Alternator

### PR DESCRIPTION
**Pull request by Orenef11**
* rolloing-upgrade.yaml - Adding new stress commands for Alternator
* upgrade_test.py:
   * fill_and_verify_alternator_db_data - The function add data and veryfy it
   * test_rolling_upgrade - Add steps to check Alternator

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/utils/alternator/data_generator.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
+import logging
```
Comment (fruch) [inline sdcm/utils/alternator/data_generator.py:1]: why we have all the random generator ? I thought we said we want specific data, and not randomely generated data for the upgrade test ?

**Comment by fruch** [comment]
@Orenef11 can you update the title of this PR and also the description , with the why this PR is done/needed
also with a link to the trello item, so if anyone joins in the review, he could understand what's going on.

**Comment by fruch** [comment]
also can you separate the changes needs a hydra new docker, since without it.
the unittest and pre-commit checking in jenkins aren't working.

**Comment by Orenef11** [comment]
> also can you separate the changes needs a hydra new docker, since without it.
> the unittest and pre-commit checking in jenkins aren't working.

I'll do it after I check that the code is working.
In any case, I wanted to separate it into two  PRs.

File: test-cases/upgrades/rolling-upgrade.yaml [hunk 2]
```diff
@@ -14,12 +14,46 @@ stress_cmd_complex_verify_read: cassandra-stress user no-warmup profile=/tmp/com
 stress_cmd_complex_verify_more: cassandra-stress user no-warmup profile=/tmp/complex_schema.yaml  ops'(read1=1,read2=1,update_static=1,update_ttl=1,update_diff1_ts=1,update_diff2_ts=1,update_same1_ts=1,update_same2_ts=1)' cl=ALL n=5000000 -mode cql3 native -rate threads=200 -pop seq=1..5000000
 stress_cmd_complex_verify_delete: cassandra-stress user no-warmup profile=/tmp/complex_schema.yaml  ops'(delete_row=1)' cl=ALL n=500000 -mode cql3 native -rate threads=200 -pop seq=1..500000
 
+# Alternator stress
+alternator_write_stress_during_entire_test: >-2
+  bin/ycsb run dynamodb -P workloads/workloada -threads 100 -p recordcount=20100200
+  -p readproportion=0 -p updateproportion=1 -p scanproportion=0 -p insertproportion=0 -p requestdistribution=uniform
+  -p fieldcount=10 -p fieldlength=512 -p operationcount=20100200
+alternator_verify_stress_after_cluster_upgrade: >-2
+  bin/ycsb run dynamodb -P workloads/workloada -threads 1000 -p recordcount=20100200
```
Comment (fruch) [inline test-cases/upgrades/rolling-upgrade.yaml:23]: why 1000 threads ? I think it might be too much for a single loader

Comment (Orenef11) [reply test-cases/upgrades/rolling-upgrade.yaml:23]: The regular stress are contains 1000 threads

**Pull request closed without merging**